app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,436 @@
+---
+title: Self-service registration for higher education institutions and their school direct partners
+description: Exploring ways to help higher education institutions register and invite their school direct partners to use the Manage service
+date: 2021-03-18
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+How might we provide a self-service journey for higher education institutions (HEIs) and their school direct partners (SDs) to register on the Manage service?
+
+## Why we are doing this
+
+Around 70 HEIs have not yet registered to use Manage. On average each of them accredits courses for 10 SDs, many of which are also not registered on Manage. We do not have contact details for all the SDs.
+
+We need to make sure that all these organisations have registered to use Manage before the UCAS service is turned off in October 2021.
+
+Adding them all manually will be a great deal of work for the transition team. We want to make the process more efficient, while not placing a significant burden on HEIs or SDs.
+
+## Requirements
+
+Before we started designing possible solutions, we spoke to members of the provendor and transition teams about their requirements.
+
+These requirements are not presented as user needs, since this work is being done primarily for the benefit of teams within DfE.
+
+### Data sharing agreement
+
+We need to:
+
+- make sure that HEIs and SDs sign a data sharing agreement (DSA)
+- record who signed the DSA
+- only show personal data to HEIs and SDs after they’ve sign the DSA, so we do not violate data protection rules
+
+### Getting contact details
+
+We need to make sure we have an up to date contact name and email address for each SD.
+
+### Inviting training providers
+
+We need to:
+
+- reduce the amount of manual work done by the transition team, to save team effort and reduce the risk of errors
+- know which HEIs are on Manage
+- know which SDs have been set up for each HEI
+- know which SDs have set up and opened courses on Manage
+
+## Provider data analysis
+
+There are 87 HEIs in the Manage database. 22 HEIs have been set up to use Manage and the remaining 65 have not.
+
+### Number of SDs connected to each HEI
+
+Using the Publish database, we looked at how many SDs are connected to each HEI.
+
+- 25% of HEIs have no SDs
+- 24% of HEIs have between 1 and 10 SDs
+- 36% of HEIs have between 11 and 20 SDs
+- 15% of HEIs have more than 20 SDs
+- Average number of SDs per HEI: 10
+- Maximum number of SDs per HEI: 33
+- Minimum number of SDs per HEI: 0
+
+If we ask an HEI user separately about each SD then on average they will see 10 pages of questions.
+
+### Number of HEIs connected to each SD
+
+- 77% of SDs have only 1 HEI
+- 14% have 2 HEIs
+- 6% have 3 HEIs
+- 3% have more than 3 HEIs
+
+This is important since if an SD has multiple HEIs, they only need to be onboarded once. The other HEIs accrediting their courses will not need to provide contact details for them.
```

Comment:
```suggestion
This is important because if an SD has multiple HEIs, they only need to be registered once. The other HEIs ratifying their courses will not need to provide contact details for them.
```